frameworks/g-cloud-9/questions/services/webChatSupport.yml

```diff
@@ -22,10 +22,13 @@ type: radios
 options:
   - label: "Yes"
     value: "yes"
+    filter_label: "Web chat"
   - label: Yes, at an extra cost
     value: yes_extra_cost
+    filter_label: "Web chat (extra cost)"
```

Comment:
Not sure if it's distinctly useful enough to have (extra cost) as a filter label. For these, we should probably just filter on yes/no. I don't expect they will want to filter based on extra costs as this point, when they don't know what the base cost is.